scripts/rng/seed_type.txt

```diff
@@ -0,0 +1,3 @@
+uint64_t

```

Comment:
should it be done by some kind of json file to match engines and their properties (such as seed types (some has several ones))?

Reply:
Done